store/local_cache_supplier_reactions.go

```diff
@@ -51,3 +51,8 @@ func (s *LocalCacheSupplier) ReactionPermanentDeleteBatch(ctx context.Context, e
 	// expire from the cache in due course.
 	return s.Next().ReactionPermanentDeleteBatch(ctx, endTime, limit)
 }
+
+func (s *LocalCacheSupplier) ReactionsBulkGetForPosts(ctx context.Context, postIds []string, hints ...LayeredStoreHint) *LayeredStoreSupplierResult {
+	// Ignoring this.
+	return s.Next().ReactionsBulkGetForPosts(ctx, postIds, hints...)
```

Comment:
@jwilander Any thoughts on whether or not we should add caching for this? My initial thoughts would be that we should have caching, but given that it's a list of IDs, it's probably relatively uncommon that you'll see any benefit from caching those results

Reply:
I wouldn't worry about caching it at this point. If it becomes an apparent bottleneck in the future we can look into adding caching or optimizing in some way